test/api/core/pluginsManager/trigger.test.js

```diff
@@ -44,6 +46,119 @@ describe('Test plugins manager trigger', () => {
       });
   });
 
+  it('should trigger hooks with before wildcard event', done => {
+    pluginsManager.plugins = [{
+      object: {
+        init: () => {},
+        hooks: {
+          'foo:before*': 'myFunc'
+        },
+        myFunc: done
+      },
+      config: {},
+      activated: true,
+      manifest: {
+        name: 'foo'
+      }
+    }];
+
+    pluginsManager.run()
+      .then(() => {
+        pluginsManager.trigger('foo:beforeBar');
+      });
+  });
+
+  it('should trigger hooks with after wildcard event', done => {
+    pluginsManager.plugins = [{
+      object: {
+        init: () => {},
+        hooks: {
+          'foo:after*': 'myFunc'
+        },
+        myFunc: done
+      },
+      config: {},
+      activated: true,
+      manifest: {
+        name: 'foo'
+      }
+    }];
+
+    pluginsManager.run()
+      .then(() => {
+        pluginsManager.trigger('foo:afterBar');
+      });
+  });
+
+  it('should trigger pipes with wildcard event', () => {
+    pluginsManager.plugins = [{
+      object: {
+        init: () => {},
+        pipes: {
+          'foo:*': 'myFunc'
+        },
+        myFunc: sinon.stub().callsArgWith(1, null, new KuzzleRequest({}))
+      },
+      config: {},
+      activated: true,
+      manifest: {
+        name: 'foo'
+      }
+    }];
+
+    pluginsManager.run()
```

Comment:
that promise must be returned, otherwise the test isn't awaited and it always passes